gemfiles/rails_7_0.gemfile

```diff
@@ -0,0 +1,22 @@
+# This file was generated by Appraisal
+
+source "https://rubygems.org"
+
+group :test do
+  gem "activejob", "~> 7.0"
+  gem "aws-sdk-core", "~> 3"
+  gem "aws-sdk-sqs"
+  gem "codeclimate-test-reporter", require: nil
+  gem "httparty"
+  gem "multi_xml"
+  gem "simplecov"
+end
+
+group :development do
+  gem "appraisal", git: "https://github.com/thoughtbot/appraisal.git"
+  gem "rubocop"
+  gem "pry", ">= 0.14.2"
+  gem "pry-byebug", ">= 3.10.1"
```

Comment:
You need a minimum, but it'd probably find it on it's own, really. It's much less of an issue in the Rails specs since each one is different, then the Ruby specs, that share a gemfile.